src/main/java/frc/robot/subsystems/gpm/StorageIndex.java

```diff
@@ -27,11 +27,14 @@ public class StorageIndex extends SubsystemBase {
    * sensor.
    */
   public StorageIndex() {
-    m_indexmotor = new CANSparkFlex(StorageIndexConstants.indexMotorID, MotorType.kBrushless);
+    // m_indexmotor = new CANSparkFlex(StorageIndexConstants.indexMotorID, MotorType.kBrushless);***
+    m_indexmotor = new TalonFX(2);
     m_indexBeamBreak = new DigitalInput(StorageIndexConstants.indexBeamBreak);
     m_indexmotor.setInverted(false);
 
-    m_indexmotor.setIdleMode(StorageIndexConstants.idleMode);
+    // m_indexmotor.setIdleMode(StorageIndexConstants.idleMode); ***
```

Comment:
Why is this commented out.

Reply:
Because someone commented it out. StorageIndex is in flux. Don't hold up a merge.